assets/css/main.scss

```diff
@@ -570,3 +570,75 @@ img.dashboard_action_icon {
   from { background-color: #ffb3c9}
   to {background-color: transparent}
 }
+
+/* ==== MAGICAL MODAL ==== */
+.modal {
+  align-items: center;
+  bottom: 0;
+  display: none;
+  flex-direction: column;
+  justify-content: center;
+  left: 0;
+  overflow: hidden;
+  position: absolute;
+  position: fixed;
```

Comment:
this overwrites the previous line... so I think you can delete the `position: absolute`

Reply:
👍